library/ix-dev/charts/minio/templates/pre-install-job.yaml

```diff
@@ -26,3 +28,5 @@ spec:
         command: ["chown", "-R", "{{ .Values.runAsUser }}:{{ .Values.runAsGroup }}", "{{ .Values.appVolumeMounts.export.mountPath }}"]
```

Comment:
When distributedMode is enabled, we hide the appVolumeMounts.
So there is no point to try to chown it anyway.